hack/scripts/install.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+CERT_MANAGER_VERSION=${CERT_MANAGER_VERSION:-v1.11.0}
+RISINGWAVE_OPERATOR_VERSION=${RISINGWAVE_OPERATOR_VERSION:-latest}
+
+function link_to_risingwave_operator_manifests() {
+  local version=$1
+
+  if [[ "${version}" == "latest" ]]; then
+    echo "https://github.com/risingwavelabs/risingwave-operator/releases/latest/download/risingwave-operator.yaml"
+  else
+    echo "https://github.com/risingwavelabs/risingwave-operator/releases/download/${RISINGWAVE_OPERATOR_VERSION}/risingwave-operator.yaml"
+  fi
+}
+
+function k8s::cert_manager::install() {
+  echo "Installing the cert-manager ${CERT_MANAGER_VERSION}..."
+  kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/"${CERT_MANAGER_VERSION}"/cert-manager.yaml >/dev/null
+}
+
+function k8s::cert_manager::_wait_validating_webhook_ca_bundle() {
```

Comment:
I found a possible alternative, that may be a bit simpler: 

```sh 
kubectl wait --for=condition=ready pod -l app.kubernetes.io/instance=cert-manager
```

Also see [SO thread](https://stackoverflow.com/questions/53536907/kubectl-wait-for-condition-complete-timeout-30s)

Reply:
Pod's readiness isn't sufficient for the cert-manager to work. We must wait for a full initialization. Here I check and wait for the CA bundle on the cert-manager's validating webhook configuration to be ready.